app/components/editor-mode-menu.js

```diff
@@ -1,6 +1,6 @@
-import Ember from 'ember';
+import Component from '@ember/component';
```

Comment:
Here again, I propose to use Glimmer component. Since here, I will not make the remark again :-)